x/wasm/keeper/stargate_bindings.go

```diff
@@ -0,0 +1,18 @@
+package keeper
+
+import (
+	"sync"
+)
+
+// StargateWhitelist keeps whitelist and its deterministic
+// response binding for stargate queries.
+//
+// The query can be multi-thread, so we have to use
+// thread safe sync.Map.
+var StargateWhitelist sync.Map
+
+// Define whitelists here as maps using 'StargateWhitelist'
+// e.x) StargateWhitelist.Store("/cosmos.auth.v1beta1.Query/Account", &authtypes.QueryAccountResponse{})
+func init() {
```

Comment:
Any suggestions what should go in here? I would prefer to not use the init() method but set/ call this from app.go.

Reply:
I'll also prefer to be something to be set by app.go or similar, since this will go unnoticed for projects that may not want to enable some of the items in this potential list.

Reply:
Should we simply remove this init method here? And then projects can simply call AcceptList.Store() or AcceptList.Load() for setter and getter for the whitelists

Reply:
@mattverse I agree you can simply remove the init method